ext/hivemq-edge-openapi-2024.8-proposal.yaml

```diff
@@ -0,0 +1,5677 @@
+    get:
+      description: Get the list of all domain tags created in this Edge instance
+      operationId: get-domain-tags
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+          description: Success
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+      summary: Get all domain tags
+      tags:
+        - Domain
+  /api/v1/management/domain/tags/schema:
+    get:
+      description: Get the data schema associated with the specified tags
+      operationId: get-tag-schemas
+      parameters:
+        - description: The list of tags to query the schema from
+          in: query
+          name: tags
+          required: true
+          schema:
+            type: array
+            items:
+              type: string
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TagSchemaList'
+          description: Success
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+      summary: Get data schemas
+      tags:
+        - Domain
+  /api/v1/management/domain/topics:
+    get:
+      description: Get the list of all topic tags created in this Edge instance
+      operationId: get-domain-topics
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: string
+          description: Success
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+      summary: Get all topic tags
+      tags:
+        - Domain
+  /api/v1/management/domain/topics/schema:
+    get:
+      description: Get the data schema associated with the specified topics
+      operationId: get-topic-schemas
+      parameters:
+        - description: The list of topics to query the schema from
+          in: query
+          name: topics
+          required: true
+          schema:
+            type: array
+            items:
+              type: string
+      responses:
+        "200":
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TagSchemaList'
+          description: Success
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+      summary: Get data schemas
+      tags:
+        - Domain
+
+  /api/v1/management/client/topic-samples:
+    get:
+      description: Get a sample of MQTT topics published by clients connected to the Edge broker over a period of time
+      operationId: get-client-topics
+      parameters:
+        - description: The time limit for the observation.
+          in: query
+          name: queryTime
+          required: true
+          schema:
+            type: integer
+            minimum: 500
+            default: 1000
```

Comment:
in millis, the default seems a bit short, there can be nothing. in seconds its far to long, we cant cache topic data for so long